worker-tasks/build.sbt

```diff
@@ -0,0 +1,7 @@
+name := "rf-worker"
+
+version := "0.1.0"
+
+scalaVersion := "2.11.1"
+
+libraryDependencies += "org.apache.spark" %% "spark-core" % "1.6.2"
```

Comment:
Addressed in https://github.com/azavea/raster-foundry/commit/00db7623e9fdc1927f3d1c682826ba1ff2817a08.